### PR TITLE
RD-4315 Updated EULA link

### PR DESCRIPTION
**Pull request by kubama**
## Description
Updated EULA link

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/2128/ - community spec only

 

## Documentation
N/A

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
                     "validationMessage": "Please provide a valid phone number, which should consists only of 4 - 20 digits"
                 },
                 "isEULA": {
-                    "label": "By registering for the Cloudify Hosted Service you agree to the <a href=\"https://cloudify.co/license\" target=\"_blank\" onclick=\"event.stopPropagation();\">Hosted Trial End User License Agreement</a>"
+                    "label": "By registering for the Cloudify Hosted Service you agree to the <a href=\"https://cloudify.co/license-community/\" target=\"_blank\" onclick=\"event.stopPropagation();\">Hosted Trial End User License Agreement</a>"
```
Comment (qooban) [inline app/translations/en.json:49]: Optional:

I'm wondering - maybe even better would be to use a value from `licenseManagement.eulaLinkCommunity` key (in `en.json`), WDYT?
Or use `EulaLink` component for "By registering for the Cloudify Hosted Service..." checkbox label?

**Review by qooban — APPROVED** [review]

**Comment by qooban** [comment]
As we are planning to release 6.3.1 ASAP, I'm merging this PR without additonal changes. They can be added in the later phase. @kubama When you're back could you create a ticket for https://github.com/cloudify-cosmo/cloudify-stage/pull/1844#discussion_r812602638 and continue (if it's quick)?

**Comment by qooban** [comment]
@kubama I'm not sure I can delete the branch, so I'm leaving it open. Please delete it when you're back.

**Comment by kubama** [comment]
@qooban Created RD-4368

**Pull request merged**